src/main/java/com/spartronics4915/frc2019/subsystems/Drive.java

```diff
@@ -756,7 +763,8 @@ public synchronized void stopLogging()
         public int right_velocity_ticks_per_100ms;
         public Rotation2d gyro_heading = Rotation2d.identity();
         public Pose2d error = Pose2d.identity();
-
+        public Rotation2d gyro_pitch = Rotation2d.identity();
+        public double[] yawPitchRoll = new double[3];
```

Comment:
This should _not_ be a member of `PeriodicIO`, but instead a member of `Drive`. Also make sure to use the correct naming convention, e.g. `mYawPitchRoll`.